src/Modules/OAuth/Domain/Settings/SettingsType.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace ForkCMS\Modules\OAuth\Domain\Settings;
+
+use ForkCMS\Core\Domain\Form\SwitchType;
+use ForkCMS\Modules\OAuth\Domain\Settings\Command\UpdateModuleSettings;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class SettingsType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add(
+                'clientId',
+                TextType::class,
+                [
+                    'label' => 'lbl.ClientId',
+                    'row_attr' => ['class' => 'col-12 col-md-6 mb-3'],
```

Comment:
Is dat nuttig om dat hier te definiëren? Zou dat niet default moeten zijn?

Reply:
Of zou de backend CSS dat niet moeten opvangen. Want lijkt me zoiets dat naar dhvelopersgemak nuttig zou zijn. En devs gaan dat vergeten toevoegen he?